src/run.py

```
@@ -91,6 +84,29 @@
+    resampling  = getattr(kwargs, 'resampling', 'bilinear')
+
+    logger.info(f"Reprojecting '{inp}' → '{out}' (target_epsg={target_epsg}, cell_size={cell_size}, resampling={resampling})")
+    wblib.reproject_to_metric(
+        input_path=inp,
+        output_path=out,
+        source_epsg=source_epsg,   
+        target_epsg=target_epsg,
+        cell_size=cell_size,
+        resampling=resampling
+    )
+    logger.info(f"Done. ")    
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)